### PR TITLE
chore(pipelined): Bump Python coverage lib version

### PR DESCRIPTION
**Pull request by koolzz**
Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Summary

- Previous library wasn't correctly reporting coverage on asyncio pipelined nosetests, increasing the version fixes the issue.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ardzoht — APPROVED** [review]
lgtm

File: lte/gateway/python/Makefile [hunk 1]
```diff
@@ -65,7 +65,7 @@ $(BIN)/nosetests: install_virtualenv
 	$(VIRT_ENV_PIP_INSTALL) -I nose==1.3.7
 
 $(BIN)/coverage: install_virtualenv
-	$(VIRT_ENV_PIP_INSTALL) "coverage==4.5.1"
+	$(VIRT_ENV_PIP_INSTALL) "coverage==6.1.2"
```
Comment (pshelar) [inline lte/gateway/python/Makefile]: do we need this specific version? without version it would use latest lib, no?

Comment (koolzz) [reply lte/gateway/python/Makefile]: I'm no expert in these but I recall a bunch of issues with specifying python libs. @tmdzk  what's your recommendation(specify with ==, >=  or not specifying at all)?

Comment (tmdzk) [reply lte/gateway/python/Makefile]: I'd recommend to do `>=` especially for the dep in the virtual env

**Review by ulaskozat — APPROVED** [review]
LGTM

**Review by pshelar — APPROVED** [review]

**Comment by themarwhal** [comment]
@koolzz I think this PR might have broken the LTE integration workflow. Seeing this when I run `make integ_test` in the test VM. The test VM uses an older version of the base image? so the version might not be available :/
```
DEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.
ERROR: Could not find a version that satisfies the requirement coverage>=6.1.2 (from lte[dev])
ERROR: No matching distribution found for coverage>=6.1.2
/home/vagrant/magma/orc8r/gateway/python/python.mk:139: recipe for target 'install_egg' failed
make[2]: *** [install_egg] Error 1
make[2]: Leaving directory '/home/vagrant/magma/lte/gateway/python'
Makefile:16: recipe for target '/home/vagrant/magma/lte/gateway/python_build' failed
make[1]: *** [/home/vagrant/magma/lte/gateway/python_build] Error 2
make[1]: Leaving directory '/home/vagrant/magma/lte/gateway/python'
Makefile:13: recipe for target '/home/vagrant/build/python/setupinteg_env' failed
make: *** [/home/vagrant/build/python/setupinteg_env] Error 2
```

**Pull request merged**